GUIDE.md

```diff
@@ -312,8 +312,8 @@ await db.access.grant('write', identity2.publicKey) // grant access to identity2
 You can create a custom access controller by implementing the `AccessController` [interface](https://github.com/orbitdb/orbit-db-access-controllers/blob/master/src/access-controller-interface.js) and adding it to the AccessControllers object before passing it to OrbitDB.
 
 ```javascript
-let AccessControllers = require('orbit-db-access-controllers')
-const AccessController = require('orbit-db-access-controllers/src/access-controller-interface')
```

Comment:
I believe we need to reference the file, even if index.js, and the .js file extension must be specified also.

```
import AccessControllers from 'orbit-db-access-controllers/index.js'
import AccessController from 'orbit-db-access-controllers/src/interface.js'
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import